--- libupnp-1.6.2/m4/rt_bool_arg_enable.m4.orig	2006-10-05 08:19:22.000000000 +0200
+++ libupnp-1.6.2/m4/rt_bool_arg_enable.m4	2007-12-15 13:53:13.234016809 +0100
@@ -41,7 +41,7 @@
 		               )])
 	test "x$enable_[$1]" != [x]m4_if([$2],[yes],[no],[yes]) dnl
 		&& enable_[]Name=[$2]
-	AC_MSG_RESULT($enable_[]Name)dnl
+	AC_MSG_RESULT($enable_[]Name)
 	AM_CONDITIONAL([ENABLE_]NAME, test x"$enable_[]Name" = xyes)
 	dnl
 	m4_popdef([NAME])dnl
