--- libupnp-1.6.2/configure.ac.orig	2007-12-11 00:04:39.000000000 +0100
+++ libupnp-1.6.2/configure.ac	2007-12-15 14:06:17.762724553 +0100
@@ -248,13 +248,6 @@
 	;;
 *)
 	echo "Using non-specific system compiler settings"
-	if test x"$enable_debug" = xyes; then
-		# AC_PROG_CC already sets CFLAGS to "-g -O2" by default
-		:
-	else
-		# add optimise for size
-		AX_CFLAGS_GCC_OPTION([-Os])
-	fi
 	;;
 esac
 
